src/utils/ingest-client/ingest-file-validator.ts

```diff
@@ -31,7 +31,7 @@ class IngestFileValidator {
      * @param fileFormat
      * @param fileName
      */
-    validateFile(fileResource: any, fileFormat: string, fileName: string) : Promise<ValidationJob> {
+    validateFile(fileResource: any, fileFormat: string, fileName: string): Promise<ValidationJob> {
```

Comment:
async function might make this a lot easier to reason about. Out of scope here, just something for the future.

Reply:
the validator codebase needs an overhaul to use the more recent and comfortable async/await syntax. I think it plays well with bluebird promises.